integration-tests/utils/templates/node_secrets.go

```diff
@@ -9,11 +9,10 @@ import (
 
 var NodeSecretsTemplate = `
 [Database]
-URL = 'postgresql://postgres:test@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable' # Required
-AllowSimplePasswords = true
+URL = 'postgresql://postgres:mysecretpassword@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable' # Required
```

Comment:
Can we store the db password here as secret as well? i.e. something like 
`postgresql://postgres:{{ .PgPass }}@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable`

Reply:
This code will be replaced in https://github.com/smartcontractkit/chainlink/pull/10067, but sure.